### PR TITLE
Adding test to verify ppc64_cpu command

### PR DESCRIPTION
**Pull request by narasimhan-v**
Signed-off-by: Narasimhan V sim@linux.vnet.ibm.com

### CODE REVIEW

**Comment by narasimhan-v** [comment]
[job_log.txt](https://github.com/avocado-framework/avocado-misc-tests/files/380827/job_log.txt)

**Comment by narasimhan-v** [comment]
@apahim can you please check this PR

File: cpu/ppc64_cpu_test.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    def setUp(self):
+        """
+        Verifies if powerpc-utils is installed, and gets current SMT value.
+        """
+        command = "uname -p"
+        if 'ppc' not in process.system_output(command, ignore_status=True):
+            self.skip("Processor is not ppc64")
+        if SoftwareManager().check_installed("powerpc-utils") is False:
+            if SoftwareManager().install("powerpc-utils") is False:
+                self.skip("powerpc-utils is not installing")
```
Comment (PraveenPenguin) [inline cpu/ppc64_cpu_test.py:41]: Just a thought why we can't package check this way 
        if not (SoftwareManager().check_installed("powerpc-utils")
                and not  SoftwareManager().install("powerpc-utils")) :
a suggestion

**Comment by PraveenPenguin** [comment]
running on Fedora24 VM guest as getting issue as SMT not enabled in system and test fails 

```
Running 'ppc64_cpu --smt=1'
[stderr] Machine is not SMT capable

Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado-38.0-py2.7.egg/avocado/core/test.py:448
Traceback (most recent call last):
  File "/root/ppc64_cpu_test.py", line 73, in test
    process.system_output("ppc64_cpu --smt=%s" % self.key, shell=True)
  File "/usr/lib/python2.7/site-packages/avocado-38.0-py2.7.egg/avocado/utils/process.py", line 1126, in system_output
    sudo=sudo)
  File "/usr/lib/python2.7/site-packages/avocado-38.0-py2.7.egg/avocado/utils/process.py", line 1033, in run
    raise CmdError(cmd, sp.result)
CmdError: Command 'ppc64_cpu --smt=1' failed (rc=255)
```

can we have also check  in Test as system SMT capable ,if  SMT capability not present ,   can  we `skip` whole test or pint a statement as`error` or can be document in README . 
My Preference is `skip` the test but at last it you .

Regards
Praveen

**Comment by narasimhan-v** [comment]
Opened a new PR #134 addressing the comments.
Closing this

**Pull request closed without merging**